documentation20/en/08.connector/01.java/docs.md

```diff
@@ -56,11 +56,11 @@ INSERT INTO test.t1 USING test.weather (ts, temperature) TAGS('beijing') VALUES(
 
 | taos-jdbcdriver version | TDengine 2.0.x.x version | TDengine 2.2.x.x version | TDengine 2.4.x.x version | JDK version |
 | ----------------------- | ------------------------ | ------------------------ | ------------------------ | ----------- |
-| 2.0.38                  | X                        | X                        | 2.4.0.12 以上            | 1.8.x       |
-| 2.0.37                  | X                        | X                        | 2.4.0.6 以上             | 1.8.x       |
-| 2.0.36                  | X                        | 2.2.2.11 以上            | 2.4.0.0 - 2.4.0.5        | 1.8.x       |
-| 2.0.35                  | X                        | 2.2.2.11 以上            | 2.3.0.0 - 2.4.0.5        | 1.8.x       |
-| 2.0.33 - 2.0.34         | 2.0.3.0 以上             | 2.2.0.0 以上             | 2.4.0.0 - 2.4.0.5        | 1.8.x       |
+| 2.0.38                  | X                        | X                        | 2.4.0.12 and above       | 1.8.x       |
```

Comment:
and later is an idiom.

https://english.stackexchange.com/questions/112257/above-or-later-when-referencing-a-range-of-versions-of-software